virt/install/get_guest_info.py

```diff
@@ -1,47 +1,84 @@
 #!/usr/bin/python2
 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+import optparse
 import os
+import shlex
 import sys
-import xmlrpclib
-import xml.dom.minidom
 import time
+import xml.dom.minidom
+import xmlrpclib
```

Comment:
Should this import be in a try except block and fall back to xmlrpc.client to be compatible with python3?

Reply:
I see now that we explicitly call python2.  I suppose upgrading this to python3 would be an entirely new PR. :-)

Reply:
Yeah. Initially, I was thinking about python2/3. But this has to be done in another PR.
We can't even have shebang inside of files if we want to be fully compatible (thank you Fedora). I will create another PR to manage it. Pretty much shebangs remove, calling explicitly interpreter for distribution and use python-six for deps.